### DISCUSSION TITLE
openHAB 4.0.0

### DISCUSSION THREAD

**Discussion by seaside1** [seaside1/jrule#154] [General]
I've started on some work on openHAB 4.0.0.
My suggestion once we get it to work is to make a hard switch to openHAB 4, so we don't have to maintain 3.x.x and 4.x.x at the same time.

Any ideas? Have you guys tested openHAB 4 yet?

**Comment by querdenker2k** [comment]
I would't support both version at the same time, but i don't know if this "works" e.g. for bugfixes in the 3.x version.
We could create a 3.x branch und work with 4.x on the main branch. If we had to fix s.th. on the 3.x branch we could do so.

No i have not tested openHAB 4 now.

**Comment by seaside1** [comment]
Main branch is 4.0.0 Closing discussion
